raster/src/main/scala/geotrellis/raster/io/geotiff/GeoTiffMultibandTile.scala

```diff
@@ -318,12 +336,114 @@ abstract class GeoTiffMultibandTile(
     tiles
   }
 
+  private def _subsetBandsFromSegments(
+    bandSequence: Seq[Int],
+    segmentIds: Traversable[Int],
+    deinterleaveBitSegment: (GeoTiffSegment, Int, Int, Int, Traversable[Int]) => Array[Array[Byte]],
```

Comment:
Thats a rough signature. Is there any way to sugar it up? Its used a couple of times, maybe type alias where we can hang a comment?